### ISSUE TITLE
Api for bot integration

### ISSUE THREAD

**Issue by crottolo** [canove/whaticket#279]
Hi, 
i tried the app and it's awesome.

I suggest a feature for send and receive the command for external interface.

Send message
webhook for receive the new message
open new ticket
close ticket
Bot interaction.

Now i want send message with workaround but with a simple rest api we can use the interface for comunicate with people, with the bot support...

thanks a lot

**Comment by KatiaSishost** [comment]
> i add in file backend /dist/libs/wbot.js at the end this:
> 
> ` var http = require('http'); var url = require('url');
> 
> //const http = require('http');
> 
> const qs = require('querystring');
> 
> http.createServer(function(req, res) { var wclient = sessions[0]; if (req.method === 'POST' && req.url === '/invia') { var body = ''; req.on('data', function(chunk) { body += chunk; }); req.on('end', function() { var dato = JSON.parse(body) //**** now you can access `data.email` and `data.password`********// wclient.sendMessage(dato.number+'@c.us', dato.message); res.writeHead(200); res.end(JSON.stringify({"send":true, "send_to": dato.number, "message": dato.message })); }); } else { res.writeHead(404); res.end(JSON.stringify({"send":false, "send_to": dato.number, "message": dato.message })); } }).listen(8081);
> 
> `
> 
> the post request is:
> 
> `{ "number": "XXXXXXXXX", "message": "Hello! \nHow are you?" }`

Olá, pode estar faltando alguma coisa?

C:\Users\User\Desktop\WhatsApp\backend\dist\libs\wbot.js:168
wclient.sendMessage(dato.number+'@c.us', dato.message);
        ^

TypeError: Cannot read property 'sendMessage' of undefined
    at IncomingMessage.<anonymous> (C:\Users\User\Desktop\WhatsApp\backend\dist\libs\wbot.js:168:9)
    at IncomingMessage.emit (events.js:327:22)
    at IncomingMessage.EventEmitter.emit (domain.js:467:12)
    at endReadableNT (internal/streams/readable.js:1327:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
[nodemon] app crashed - waiting for file changes before starting...

**Comment by Gecko2087** [comment]
Thank you very much for your contribution.
I have some questions. If I modify this on my server, will I be able to send messages by get? For example by a url like this: whaticket.myserver.com/send?number=XXXXXXXXX

And these messages cause a new ticket to be created?

**Comment by rodrigofsantos** [comment]
Sim bastaria para isso criar um host virtual exemplo mass.myserver.com apontando para a porta do serviço ai funcionaria conforme você precisa !

**Comment by crottolo** [comment]
> Thank you very much for your contribution. I have some questions. If I modify this on my server, will I be able to send messages by get? For example by a url like this: whaticket.myserver.com/send?number=XXXXXXXXX
> 
> And these messages cause a new ticket to be created?

My solution require post payload to /invia endpoint.


If you send a message, automatically start a new ticket in queue.

if you have a solution to close automatically the ticket it’s been accepted 😊

**Comment by rodrigofsantos** [comment]
If you send a message, automatically start a new ticket in queue.
R: Sim
if you have a solution to close automatically the ticket it’s been accepted 
R: Eunão sei nunca tentei mas e possivel !!!

**Comment by crottolo** [comment]
If you 

> If you send a message, automatically start a new ticket in queue.
> R: Sim
> if you have a solution to close automatically the ticket it’s been accepted
> R: Eunão sei nunca tentei mas e possivel !!!

If you have an example is awesome 🤩

**Comment by IsraelFernando** [comment]
Exactly!!! It will help a lot @rodrigofsantos

**Comment by KatiaSishost** [comment]
ola! e poder enviar vários arquivos?
